modules/zuora/src/orders/orderRequests.ts

```diff
@@ -30,7 +30,7 @@ export type OrderRequest = AccountOrderRequest & {
 		subscriptionNumber?: string;
 		orderActions: OrderAction[];
 		customFields?: {
```

Comment:
as per below comment
```suggestion
		customFields: {
```